src/lib/shared/dialog.ts

```diff
@@ -129,6 +129,14 @@ export interface DialogPhrases {
 
   /** English: "View" */
   "activateIntegrationDialog.viewButton": SimplePhrase;
+
+  /** English: "New integration (%{id})" */
+  "activateIntegrationDialog.nameValue": ComplexPhrase<{
+    id: string;
+  }>;
```

Comment:
This shouldn't contain the uuid. That's purely for name uniqueness constraints and shouldn't be exposed to the user.

Reply:
@bkegley removed!